auth-api/migrations/versions/2024_09_20_aa74003de9d8_.py

```diff
@@ -147,7 +147,7 @@ def upgrade():
         sa.Column("modified_by_id", sa.Integer(), autoincrement=False, nullable=True),
         sa.Column("version", sa.Integer(), autoincrement=False, nullable=False),
         sa.Column("changed", sa.DateTime(), nullable=True),
-        sa.PrimaryKeyConstraint("id", "version"),
+        sa.PrimaryKeyConstraint("id", "changed"),
```

Comment:
new migration to handle this?

Reply:
already changed the key manually. wanted to verify it in dev/test first

Reply:
looks like it will remain busted on my computer until I change it or dump the db then haha

Reply:
sorry for the inconvenience, that migration file already had a few updates, thought no one will notice

Reply:
all good haha